### PR TITLE
[4][com_installer] unable to update an extension

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
fix the extension update

### Testing Instructions
install  com_patchtester  4.0.0  and then try to update to com_patchtester 4.1.0

### Actual result BEFORE applying this Pull Request
error

### Expected result AFTER applying this Pull Request
update goes ok

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on ff5ce8f70fa5b12e2d6e3e26a9615e9d009ba061<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33497">issues.joomla.org/tracker/joomla-cms/33497</a>.</sub>

File: administrator/components/com_installer/src/Model/UpdateModel.php [hunk 1]
```diff
@@ -22,6 +22,7 @@
 use Joomla\CMS\Plugin\PluginHelper;
 use Joomla\CMS\Updater\Update;
 use Joomla\CMS\Updater\Updater;
+use Joomla\CMS\Table\Table;
```
Comment (joomdonation) [inline administrator/components/com_installer/src/Model/UpdateModel.php:25]: Should we move this use statement above `use Joomla\CMS\Updater\Update;`  (alphabet order) ?

**Comment by richard67** [comment]
It seems this PR has become obsolete with the following commit: https://github.com/joomla/joomla-cms/commit/8a1d803f59a669dfe4574867bf5e5b7d751f6d91

**Comment by joomdonation** [comment]
Ah, with that commit, then this PR is not needed, then

**Comment by richard67** [comment]
Closing as meanwhile obsolete due to recent commit https://github.com/joomla/joomla-cms/commit/8a1d803f59a669dfe4574867bf5e5b7d751f6d91 in the 4.0-dev branch.

**Pull request closed without merging**